recipe/meta.yaml

```diff
@@ -1,40 +1,45 @@
 {% set name = "probableparsing" %}
 {% set version = "0.0.1" %}
-{% set hashtype = "md5" %}
-{% set hash = "960a1fab176276070111f1f9b2080cca" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  {{ hashtype }}: {{ hash }}
+  sha256: 8114bbf889e1f9456fe35946454c96e42a6ee2673a90d4f1f9c46a406f543767
 
 build:
-    noarch: python
-    number: 1000
-    script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv 
 
 requirements:
-    build:
-        - python
-        - setuptools
-        - future
-    run:
-        - python
-        - future
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python
+
 test:
-    imports:
-        - probableparsing
+  imports:
+    - probableparsing
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
-    home: https://github.com/datamade/probableparsing
-    license: MIT
-    license_family: MIT
-    summary: Common methods for probable parsing
+  home: https://github.com/datamade/probableparsing
+  license: MIT
+  license_family: MIT
+  license_url: https://github.com/datamade/probableparsing/blob/master/LICENSE
```

Comment:
does the upstream license need to be included in the conda package? I thought a link was ok (otherwise license_url wouldn't exist) or am I missing something?

Reply:
The MIT license states:
>The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

Reply:
ah. thanks for the explanation.

Reply:
In general, it's highly recommended to always include the license file as is no matter wha the license is. It's the safest thing to do.

Reply:
ok, good to know.

Reply:
embedded in the `recipe` folder